### DISCUSSION TITLE
SignalStore and Redux Pattern

### DISCUSSION THREAD

**Discussion by anthony-b-dev** [ngrx/platform#4191] [Q&A] [answered]
I've been trying to consume as much info about the signalstore as possible but I can't seem to wrap my head around how it will implement the redux pattern. Seems like ngrx store had a very clear separation of responsibilities and unidirectional data flow, but the signal store seems to blur that line. 

Any advice would be greatly appreciated!

**Comment by rainerhahnekamp** [comment, marked as answer]
At the moment, it looks like it is not going to happen: https://github.com/ngrx/platform/issues/4031

We are currently working on a redux extension as a community project: https://github.com/angular-architects/ngrx-toolkit.

You can try it and give feedback or contribute.

**Reply by anthony-b-dev** [reply]
Ok this confirms that I wasn't missing something. Thanks! Great work on the library so far! Thanks for all the hard work!
